src/query/storages/parquet/src/parquet_table/partition.rs

```diff
@@ -114,7 +114,7 @@ impl ParquetTable {
                 .map(|f| (f.path.clone(), f.size))
                 .collect::<Vec<_>>(),
             None => if self.operator.info().can_blocking() {
-                self.files_info.blocking_list(&self.operator, false)
+                self.files_info.blocking_list(&self.operator, false, None)
```

Comment:
I can't quite remember it:  when will the code switch to the "None=>" arm (maybe unreachable)?  
to make sure is ok to omit max_file here

Reply:
Oh, I recall now， select stage （no copy ）